src/actions/app.js

```diff
@@ -157,8 +157,8 @@ export function setupInitialUrlState(
     // load process.
     withHistoryReplaceStateSync(() => {
       dispatch(updateUrlState(urlState));
-      dispatch(urlSetupDone());
       dispatch(finalizeProfileView());
+      dispatch(urlSetupDone());
```

Comment:
This wasn't really needed to fix the problem, but this looks more correct: indeed finalizeProfileView will still change the urlState possibly.

Reply:
Yeah, this indeed looks more correct.